proxy/bootstrap/src/main/java/org/apache/shardingsphere/proxy/Bootstrap.java

```diff
@@ -52,6 +54,7 @@ public static void main(final String[] args) throws IOException, SQLException {
         YamlProxyConfiguration yamlConfig = ProxyConfigurationLoader.load(bootstrapArgs.getConfigurationPath());
         int port = bootstrapArgs.getPort().orElseGet(() -> new ConfigurationProperties(yamlConfig.getServerConfiguration().getProps()).getValue(ConfigurationPropertyKey.PROXY_DEFAULT_PORT));
         List<String> addresses = bootstrapArgs.getAddresses();
+        checkPort(addresses, port);
```

Comment:
this method only solved the issue of port, if other unknown exceptions have occurred at proxy.start(), the proxy will not exit, and the jobs or other resources will not be released

Reply:
After closing the ejob that collects metadata, if an exception occurs after startup, the process will be exited.